doc/source/glossary.rst

```diff
@@ -95,54 +95,55 @@ Glossary
 
 
    along an axis
-       Axes are defined for arrays with more than one dimension.  A
-       2-dimensional array has two corresponding axes: the first running
-       vertically downwards across rows (axis 0), and the second running
-       horizontally across columns (axis 1).
 
-       Many operations can take place along one of these axes.  For example,
-       we can sum each row of an array, in which case we operate along
-       columns, or axis 1::
+       The result of an operation along an :term:`axis` X is an array in which X
+       disappears. This can surprise new users expecting the opposite.
```

Comment:
This description is inaccurate:
```
>>> x = np.arange(12).reshape(3, 4)
>>> x.shape
(3, 4)
>>> np.sort(x, axis=0).shape  # sort along axis 0
(3, 4)  # axis didn't disappear!
```

It would be accurate as
```suggestion
       The result of an reduction along an :term:`axis` X is an array in which X
       disappears. This can surprise new users expecting the opposite.
```
but I suspect you want this glossary entry to describe the more general case.

Reply:
Thanks for pointing this out. Alternative in 511d769 .